main.go

```diff
@@ -107,6 +107,7 @@ func cmd() *cobra.Command {
 				case "all":
 					context.UpgradeBridgeVersion = true
 					context.UpgradeProviderVersion = true
+					context.UpgradeSdkVersion = true
```

Comment:
I'm not sure why we weren't previously opting sdk upgrades into `"all"`

Reply:
We decided we didn't want to default pushing the sdk version past what the bridge requires. The bridge depends on too many internals of pulumi/pkg and this often breaks.

It would be great if you could add a comment explaining as such.

```suggestion
```

Reply:
For example, if you did the same upgrade several hours later and depended on pulumi/pkg v3.76.0 it would fail to compile against the bridge.